sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample4_MultivariateDetect.cs

```diff
@@ -144,7 +144,10 @@ private MultivariateDetectionResult BatchDetect(AnomalyDetectorClient client, Ur
             try
             {
                 Console.WriteLine("Start batch detect...");
-                MultivariateBatchDetectionOptions request = new MultivariateBatchDetectionOptions(datasource, 10, startTime, endTime);
+                MultivariateBatchDetectionOptions request = new MultivariateBatchDetectionOptions(datasource, startTime, endTime)
+                {
+                    TopContributorCount = 10
+                };
```

Comment:
as a sample, we should not use anything internal.
This ctor we previously used is internal, and the MRW feature changes the signature of internal ctors, therefore we have a build failure here.
Here I change it to use the public ctor instead.